src/ray/stats/stats.h

```diff
@@ -89,6 +89,12 @@ static inline void Init(const TagsType &global_tags, const int metrics_agent_por
     exporter = exporter_to_use;
   }
 
+  // Set interval.
+  StatsConfig::instance().SetReportInterval(
+      absl::Seconds(RayConfig::instance().metrics_report_interval_seconds()));
+  StatsConfig::instance().SetHarvestInterval(
+      absl::Seconds(RayConfig::instance().metrics_report_interval_seconds() / 2));
```

Comment:
If `RayConfig::instance().metrics_report_interval_seconds() == 1` then `RayConfig::instance().metrics_report_interval_seconds() / 2 == 0` which is dangerously low. Maybe cap this to 1s with `std::max(RayConfig::instance().metrics_report_interval_seconds() / 2, 1)`?

Reply:
ah, really good catch. Will fix.